tests/unit/clients/test_s3_client.py

```diff
@@ -19,10 +19,6 @@ def fixture_conn():
 
 
 class TestS3Client:
-    def test_invalid_scheme(self, register_handler):
```

Comment:
The client still checks validity though?

Reply:
I'll add that in the new registers, still need to think the best way to do this